versions/3.0.4.md

```diff
@@ -3495,3 +3497,53 @@ Version   | Date       | Notes
 1.2       | 2014-03-14 | Initial release of the formal document.
 1.1       | 2012-08-22 | Release of Swagger 1.1
 1.0       | 2011-08-10 | First release of the Swagger Specification
+
+## <a name="usingRFC6570Implementations"></a>Appendix C: Using RFC6570 Implementations
+
+Serialization is defined in terms of RFC6570 URI Templates in two scenarios:
+
+Object | Condition
+------ | ---------
+[Parameter Object](#parameterObject) | When `schema` is present
+[Encoding Object](#encodingObject) | When encoding for `application/x-www-form-urlencoded` and any of `style`, `explode`, or `allowReserved` are used
+
+Implementations of this specification MAY use an implementation of RFC6570 to perform variable expansion, however, some caveats apply.
+
+### Equivalences Between Fields and RFC6570 Operators
+
+Certain field values translate to RFC6570 operators (or lack thereof):
+
+field | value | operator
+----- | ----- | --------
+style | simple | _n/a_
+style | matrix | `;`
+style | label | `.`
+style | form | `?`
+allowReserved | `false` | _n/a_
+allowReserved | `true` | `+`
+explode | `false` | _n/a_
```

Comment:
RFC 6570 refers to explode as a modifier because it is a suffix added to the variable as opposed to an operator which appears as a prefix.

Reply:
Ah, yes.  I was just going by the characters, not the placement, but you're right that it should be more clear.

Reply:
I changed the headings to make things more clear in the latest reworking of this PR.